docs/docs/configuration/db.md

```diff
@@ -1,126 +1,153 @@
 
-### Skip update of vulnerability DB
-If you want to skip downloading the vulnerability database, use the `--skip-db-update` option.
+## External Services
 
-```
-$ trivy image --skip-db-update python:3.4-alpine3.9
-```
+In addition to the above, some specific scanning features might connect to external services, and have different connectivity requirements and configuration options. This document discusses only Trivy's own databases, but for your convenience here are use cases where external services are involved:
 
-<details>
-<summary>Result</summary>
+- [Java vulnerability scanning](../coverage/language/java.md).
 
-```
-2019-05-16T12:48:08.703+0900    INFO    Detecting Alpine vulnerabilities...
-
-python:3.4-alpine3.9 (alpine 3.9.2)
-===================================
-Total: 1 (UNKNOWN: 0, LOW: 0, MEDIUM: 1, HIGH: 0, CRITICAL: 0)
-
-+---------+------------------+----------+-------------------+---------------+--------------------------------+
-| LIBRARY | VULNERABILITY ID | SEVERITY | INSTALLED VERSION | FIXED VERSION |             TITLE              |
-+---------+------------------+----------+-------------------+---------------+--------------------------------+
-| openssl | CVE-2019-1543    | MEDIUM   | 1.1.1a-r1         | 1.1.1b-r1     | openssl: ChaCha20-Poly1305     |
-|         |                  |          |                   |               | with long nonces               |
-+---------+------------------+----------+-------------------+---------------+--------------------------------+
-```
+## Locations
+
+Following are official locations of Trivy databases:
+
+| Registry | Image Address | Link
+| --- | --- | ---
+| GHCR | `ghcr.io/aquasecurity/trivy-db` | <https://ghcr.io/aquasecurity/trivy-db>
+| | `ghcr.io/aquasecurity/trivy-java-db` | <https://ghcr.io/aquasecurity/trivy-java-db>
+| | `ghcr.io/aquasecurity/trivy-checks` | <https://ghcr.io/aquasecurity/trivy-checks>
+| Docker Hub | `aquasec/trivy-db` | <https://hub.docker.com/r/aquasec/trivy-db>
+| | `aquasec/trivy-java-db` | <https://hub.docker.com/r/aquasec/trivy-java-db>
+| AWS ECR | `public.ecr.aws/aquasecurity/trivy-db` | <https://gallery.ecr.aws/aquasecurity/trivy-db>
+| | `public.ecr.aws/aquasecurity/trivy-java-db` | <https://gallery.ecr.aws/aquasecurity/trivy-java-db>
+
+ VEX Hub is fetched from VEX Hub GitHub Repository directly: <https://github.com/aquasecurity/vexhub>.
+
+### Automatic fallback
+
+Trivy will attempt to pull images from the official registries in the order specified. In case of a transient errors (e.g. status 429 or 5xx), Trivy will fall back to alternative registries in the order specified.  
+You can specify additional alternative repositories as explained in the [configuring database locations section](#locations).
+
+The Checks Database registry location option does not support fallback through multiple options. This is because in case of a failure pulling the trivy-chekcs DB, Trivy will use the embedded checks as a fallback.
```

Comment:
We call it "checks bundle" in all other places. Do we want to rename it officially?

Reply:
I'm fine with it.

Reply:
@simar7 I think you named it "checks bundle." What do you think?

Reply:
Yeah checks bundle is good with me.

Reply:
@itaysk is calling it "checks database" in this PR. We should be consistent in what we call it within the project. Either is fine for me.

- Checks Bundle
- Checks Database

Note we already have a flag `--checks-bundle-repository` and need to rename it to `--checks-db-repository` if we'll go with checks database.

```
      --checks-bundle-repository string   OCI registry URL to retrieve checks bundle from (default "ghcr.io/aquasecurity/trivy-checks:1")
```

Reply:
I'm also fine with either.